src/main/java/org/javarosa/xml/TreeElementParser.java

```diff
@@ -26,7 +34,7 @@ public TreeElementParser(KXmlParser parser, int multiplicity, String instanceId)
 
     @Override
     public TreeElement parse() throws InvalidStructureException, IOException,
-            XmlPullParserException, UnfullfilledRequirementsException {
+        XmlPullParserException {
```

Comment:
Please just put back this even if it's not thrown `UnfullfilledRequirementsException`. that's the hard part of legacy code 😄